src/main/java/com/google/step/snippet/external/StackOverflowClient.java

```diff
@@ -39,9 +39,19 @@
   private static final String ANSWER_ID_PARAMETER = "answer_id";
   // Set 200 to be the maximum length of description for MVP.
   private static final int MAX_DESCRIPTION_LENGTH = 200;
+  private String CSE_ID;
```

Comment:
I like to put a newline between static and instance variables to visually separate them.